test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -78,12 +80,24 @@ describe('Client Side Encryption (Legacy)', function () {
     return testContext.setup(this.configuration);
   });
 
-  generateTopologyTests(testSuites, testContext, spec => {
-    return !SKIPPED_TESTS.has(spec.description);
+  generateTopologyTests(testSuites, testContext, ({ description }) => {
+    if (SKIPPED_TESTS.has(description)) {
+      return false;
+    }
+    if (isServerless) {
+      // TODO(NODE-4730): Fix failing csfle tests against serverless
+      return [
+        'BypassQueryAnalysis decrypts',
+        'encryptedFieldsMap is preferred over remote encryptedFields'
+      ].includes(description);
+    }
+    return true;
   });
 });
 
 describe('Client Side Encryption (Unified)', function () {
   installNode18DNSHooks();
-  runUnifiedSuite(loadSpecTests(path.join('client-side-encryption', 'tests', 'unified')));
+  runUnifiedSuite(loadSpecTests(path.join('client-side-encryption', 'tests', 'unified')), () =>
```

Comment:
changes in this file are necessary because we don't want to run the unified tests on serverless.

I considered breaking them into two separate files but decided against it for two reasons:

- our pattern is to keep legacy / unified tests together in the same file
- I wasn't sure how many places we ran just this file, and consequently how many places I might need to update if I broke them apart